modules/scaleway/cert-manager.tf

```diff
@@ -15,6 +15,13 @@ locals {
       acme_http01_enabled       = false
       acme_http01_ingress_class = "nginx"
       acme_dns01_enabled        = false
+      acme_dns01_provider       = ""
+      acme_dns01_hosted_zone_id = ""
+      acme_dns01_aws_secret     = ""
+      acme_dns01_aws_access_key_id  = ""
+      acme_dns01_aws_access_key_secret = ""
+      acme_dns01_region         = ""
+      cluster_issuer_assume_role_arn = ""
```

Comment:
`cluster_issuer_assume_role_arn` is not used anywhere

Reply:
removed, thanks